src/test/auto-completer_test.ts

```diff
@@ -474,6 +474,79 @@ suite('AutoCompleter', () => {
               'polymer/element-with-databinding.html', internalPropUsePosition),
           databindingCompletions);
     });
+
+
+    test(`autocomplete css custom properties`, async() => {
+      const {client} = await createTestEnvironment();
+      await client.openFile('index.html', `
+        <style>
+          div {
+            --foo: red;
+            --bar: var(--baz);
+            --fizz: {--floo: var(--flub);};
+          }
+        </style>
+      `);
+      const declaractionPosition = {line: 3, column: 16};
+      const usePosition = {line: 4, column: 27};
```

Comment:
Similar to above, in vscode this location does not exist, the column goes up to 26 
![screen shot 2017-12-06 at 10 12 22 am](https://user-images.githubusercontent.com/6173664/33677522-e8b0148a-da6d-11e7-9079-af284be3caf1.png)

Reply:
Tools report 1-indexed, code uses 0-indexed